loafwallet/src/Strings/it.lproj/Localizable.strings

```diff
@@ -974,7 +974,7 @@
 "Send.UnstoppableDomains.simpleplaceholder" = "Inserisci dominio";
 
 /* UDSystemError */
-"Send.UnstoppableDomains.udSystemError" = "Problema di ricerca nel sistema. [Errore: %2$d]";
+"Send.UnstoppableDomains.udSystemError" = "Sistem arama sorunu. [Hata:%2$d]";
```

Comment:
Sorry, I know this is mega-minor, but just to keep it consistent, could we add one space after Hata:?
`"Send.UnstoppableDomains.udSystemError" = "Sistem arama sorunu. [Hata: %2$d]";`

Reply:
Oh hell yeah.  Thanks @Iferencak